SentinelOne/cloud_funnel2.0/_meta/smart-descriptions.json

```
@@ -977,5 +1197,110 @@
         "field": "deepvisibility.process.storyline_id"
       }
     ]
+  },
+  {
+    "value": "{action.properties.TargetDomainName}\\{action.properties.TargetUserName} logged on to {host.name} (LogonType {action.properties.LogonType})",
+    "relationships": [
+      {
+        "source": "action.properties.TargetUserName",
+        "target": "host.name",
+        "type": "logged on to"
+      }
+    ],
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "Login"
+      },
+      {
+        "field": "action.outcome",
+        "value": "success"
+      }
+    ]
+  },
+  {
+    "value": "{action.properties.TargetDomainName}\\{action.properties.TargetUserName} logged on to {host.name} from IP {source.ip} (LogonType {action.properties.LogonType})",
+    "relationships": [
+      {
+        "source": "action.properties.TargetUserName",
+        "target": "host.name",
+        "type": "logged on to"
+      },
+      {
+        "source": "action.properties.TargetUserName",
+        "target": "source.ip",
+        "type": "connected from"
+      }
+    ],
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "Login"
+      },
+      {
+        "field": "action.outcome",
+        "value": "success"
+      },
+      {
+        "field": "source.ip"
+      }
+    ]
+  },
+  {
+    "value": "{action.properties.TargetDomainName}\\{action.properties.TargetUserName} failed to log on to {host.name} from IP {source.ip} (LogonType {action.properties.LogonType})",
+    "relationships": [
+      {
+        "source": "action.properties.TargetUserName",
+        "target": "host.name",
+        "type": "failed to log on to"
+      },
+      {
+        "source": "action.properties.TargetUserName",
+        "target": "source.ip",
+        "type": "connected from"
+      }
+    ],
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "Login"
+      },
+      {
+        "field": "action.outcome",
+        "value": "failure"
+      },
+      {
+        "field": "source.ip"
+      }
```

Some conditions are missing.
```suggestion
      },
      {
        "field": "action.properties.TargetDomainName"
      },
      {
        "field": "action.properties.TargetUserName"
      },
      {
        "field": "action.properties.LogonType"
      },
      {
        "field": "host.name"
      }
```